apps/wallet/src/ui/app/shared/faucet/request-button/index.tsx

```diff
@@ -16,40 +18,90 @@ import type { ButtonProps } from '_app/shared/button';
 
 import st from './RequestButton.module.scss';
 
+//TODO:  Remove post testnet
+const TESTNET_DISCORD_LINK =
+    'https://discord.com/channels/916379725201563759/1037811694564560966';
+
 type FaucetRequestButtonProps = {
     mode?: ButtonProps['mode'];
     trackEventSource: 'home' | 'settings';
 };
 
+function FaucetDiscordLink({
+    mode = 'primary',
+    trackEventSource,
+}: {
+    mode?: ButtonProps['mode'];
+    trackEventSource: 'home' | 'settings';
+}) {
+    return (
+        <ExternalLink
+            className={cl(
```

Comment:
We need to change our Button component (I will probably need to do it for the multi account work so not for this PR) but I think we should make the buttons (this and the faucet one) look the same because now it looks a bit weird

| discord link | faucet button |
| - | - |
| <img width="388" alt="Screenshot 2023-02-03 at 11 34 57" src="https://user-images.githubusercontent.com/10210143/216594708-d7df5a04-255c-4baf-9936-b6264ccaf488.png"> | <img width="388" alt="Screenshot 2023-02-03 at 11 34 46" src="https://user-images.githubusercontent.com/10210143/216594824-cccaebf2-e772-42c8-97cf-a716616c649d.png"> |
| <img width="388" alt="Screenshot 2023-02-03 at 11 34 30" src="https://user-images.githubusercontent.com/10210143/216594868-b743f40d-7c52-4d49-a8e1-05a5e13b572b.png"> | <img width="388" alt="Screenshot 2023-02-03 at 11 34 23" src="https://user-images.githubusercontent.com/10210143/216594931-e04e4e80-7993-4777-bf59-f765e0cfcd96.png"> |

Reply:
I switch the color in the scss file to match the design. Although I don't secondary name is weird for outline button